crates/oxc_linter/src/snapshots/no_async_promise_executor.snap

```diff
@@ -0,0 +1,23 @@
+---
+source: crates/oxc_linter/src/tester.rs
+expression: no_async_promise_executor
+---
+
+  ⚠ eslint(no-async-promise-executor): Promise executor functions should not be `async`.
+   ╭─[no_async_promise_executor.tsx:1:1]
+ 1 │ new Promise(async function foo(resolve, reject) {})
+   ·             ──────────────────────────────────────
```

Comment:
We should be able to point to this `async`: `new Span(start, start+5)`

Reply:
> We should be able to point to this `async`: `new Span(start, start+5)`

I had initially intended on doing it.
However, I found that `async` lacks a span. 
Since `async` is a single token, it can indeed be calculated. Truly a clever approach.